atomic_reactor/plugins/exit_koji_import.py

```diff
@@ -135,6 +136,10 @@ def set_help(self, extra, worker_metadatas):
                         # They are all None
                         extra['image']['help'] = None
 
+    def set_brew_versions(self, extra, worker_metadatas):
+        build_id, brew_version = self.workflow.postbuild_results.get(PulpPullPlugin.key)
```

Comment:
You wanna read from `worker_metadatas` right?

`pulp_pull` runs in worker build, and `koji_import` in orchestrator build.
`self.workflow.postbuild_results.get(PulpPullPlugin.key)` will always return None, I think.